packages/suite/src/views/dashboard/components/AssetsView/components/AssetTableRowGrid.tsx

```diff
@@ -0,0 +1,23 @@
+import styled, { css } from 'styled-components';
+import { ArrowIcon } from './AssetInfo';
+
+export const AssetTableRowGrid = styled.div`
+    display: grid;
+    overflow: hidden;
+    grid-template-columns: 2fr 2fr 1fr 1fr 1fr;
+    border-radius: 8px;
+
+    ${({ onClick }) =>
+        onClick !== undefined
+            ? css`
+                  cursor: pointer;
+                  :hover {
+                      background: #d9d9d933;
```

Comment:
I would prefer use some color from our there here

Reply:
Yea, I just took it from the figma. Not sure how to organize color in best way. In our theme there is just ton of them.

Reply:
done